custom/icds_reports/utils/aggregation_helpers/distributed/thr_forms_child_health.py

```diff
@@ -44,7 +44,7 @@ def aggregation_query(self):
             %(month)s AS month,
             child_health_case_id AS case_id,
             MAX(timeend) over w AS latest_time_end_processed,
-            SUM(days_ration_given_child) over w AS days_ration_given_child
+            CASE WHEN SUM(days_ration_given_child) over w < 32767 THEN SUM(days_ration_given_child) over w ELSE 32767 END AS days_ration_given_child
```

Comment:
@calellowitz  did not the agg failed because the value was 32000 which is still lower than 32767? in this it will still fail right? Am I missing something silly?

Reply:
these people are sick!.
But not the fix. it looks good.